documentation/optee_design.md

```diff
@@ -665,16 +665,38 @@ instead. When the page is mapped new data is copied from backing store and
 the hash of the page is verified. If it's OK the pager returns from the
 exception to resume the execution.
 
-# 9. Cryptographic abstraction layer
+# 9. Memory objects
+
+A memory object, MOBJ, describes a piece of memory. The interface provided
+is mostly abstract when it comes to using the MOBJ to populate translation
+tables etc.
+
+There's different kinds of MOBJs describing:
+- physically contiguous memory
+  - created with mobj_phys_alloc()
+- virtual memory
+  - one instance with the name mobj_virt available
+  - spans the entire virtual address space
```

Comment:
No, it's the entire virtual address space. Typically used when a user TA is invoking a static TA.